.eslintrc

```diff
@@ -18,7 +18,6 @@
         "import/default": "error",
         "import/no-absolute-path": "error",
         "import/no-self-import": "error",
-        "import/no-cycle": "error",
```

Comment:
Do we really want to get rid of this?

Reply:
I'm a little concerned about this as well, but since we're still prevent direct imports between packages I think we're okay.